plugins/module_utils/schema.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Akini Ross (@akinross) <akinross@cisco.com>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
```

Comment:
I think it needs to be GPL because some other files are GPL.

Reply:
I kept it the same as the mso module_utils: https://github.com/CiscoDevNet/ansible-mso/blob/master/plugins/module_utils/mso.py. But please confirm which one should be used.

Reply:
changed it to gpl